packages/app/cypress/e2e/specs.cy.ts

```diff
@@ -416,19 +424,11 @@ describe('App: Index', () => {
         cy.findAllByLabelText(defaultMessages.createSpec.e2e.importEmptySpec.inputPlaceholder)
         .as('enterSpecInput')
 
-        cy.get('@enterSpecInput').clear().type(getPathForPlatform('src/e2e/MyTest.spec.j'))
-        cy.intercept('mutation-EmptyGenerator_MatchSpecFile', (req) => {
-          if (req.body.variables.specFile === getPathForPlatform('src/e2e/MyTest.spec.jx')) {
-            req.on('before:response', (res) => {
-              res.body.data.matchesSpecPattern = true
-            })
-          }
-        })
```

Comment:
This intercept was really indirect. I updated the spec to just input a spec that matches the pattern so that the expected warning is presented.